src/app/shared/error-toasters/form-error/form-error.component.html

```diff
@@ -16,6 +16,19 @@
 </div>
 
 <ng-template #generalMessage>
-  <div style="max-width: 300px; display: inline-block">{{ 'Error code ' + data.status + ': ' + data.message }}</div>
-  <button style="float: right; margin-top: 8px" mat-button color="accent" (click)="snackBarRef.dismiss()">Close</button>
+  <div style="display: flex; align-items: center; justify-content: space-between">
```

Comment:
@chunweii is it possible to shift the styling to a separate `.css` file so that we won't need to pollute our html elements with the necessary styling! What are your thoiughts? cc @dingyuchen

Reply:
I'm ok with either as the component itself is quite small.